assets/stylesheets/shared/functions/_z-index.scss

```diff
@@ -188,6 +188,9 @@ $z-layers: (
 		'.media-library__list-item-spinner': 20,
 		'.media-library__list-item-edit': 20
 	),
+	'.media-library': (
+		'.section-nav .search.is-expanded-to-container': 20,
```

Comment:
Just fyi that you can use this Chrome extension to figure out the parent context https://chrome.google.com/webstore/detail/z-context/jigamimbjojkdgnlldajknogfgncplbh

So a few things here:
1. The parent stacking context is `.section-nav` to make this more explicit for the media library I'd recommend using:
```jsx
'.media-library .section-nav': ( '.search.is-expanded-to-container' : 1,
```
2. Why is this value 20? It doesn't appear like we need to beat any other items in the current stacking context so I think 1 works here just fine. Let me know if you had some corner cases in mind.

As an aside, we also have a z-index on the search icon, but it does nothing since the position is static.
![screen shot 2017-03-08 at 10 37 33 am](https://cloud.githubusercontent.com/assets/1270189/23718210/745f1936-03eb-11e7-9115-f05d663d6833.png)